prow/plugins/plugin-config-documented.yaml

```diff
@@ -588,18 +600,8 @@ repo_milestone:
         maintainers_friendly_name: ' '
         maintainers_team: ' '
 require_matching_label:
-  - # Branch is the branch ref of PRs that this config applies to.
-    # This field is only valid if `prs: true` and may be omitted to apply this
-    # config across all branches in the repo or org.
-    branch: ' '
-
-    # GracePeriod is the amount of time to wait before processing newly opened
-    # or reopened issues and PRs. This delay allows other automation to apply
-    # labels before we look for matching labels.
-    # Defaults to '5s'.
+  - branch: ' '
```

Comment:
I tried embedding in the structs present in `config.go` to try and reuse code, but I guess the documentation messes up if I do that. Maybe go with adding a new field/duplicating fields?

Reply:
IIUC you can run hack/gen-prow-documented to re-generate the documentation using the new config